etc/env/example-production.env

```diff
@@ -2,8 +2,8 @@
 # - - -
 
 # site options
-CANDIG_MODULES=logging keycloak vault redis postgres htsget candig-api tyk opa federation candig-ingest candig-data-portal
     # drs-server wes-server monitoring
+CANDIG_MODULES=logging keycloak vault redis tyk opa postgres candig-api htsget rnaget federation candig-data-portal
```

Comment:
With the removal of candig-ingest, there's no `/ingest/user/me` endpoint, which is required for the frontend to know the details of the current user. This could be moved to candig-api, but I can't accept this PR until it exists somewhere or this will be a regression.